packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -543,7 +605,7 @@ const extractTemplateLiteral = (node: t.TemplateLiteral, meta: Metadata): CSSOut
 
   css.push({ type: 'unconditional', css: literalResult });
 
-  // Deals with Conditional CSS Rules
+  // Deals with Conditional CSS Rules from Logical Expressions
   node.expressions.forEach((prop) => {
```

Comment:
Can this logic be moved into the main looping logic above? I can't quite remember the reasoning for having it here

Reply:
I'm honestly not very certain about why it isn't in the main loop, but there is already a logic path in the main loop that would conflict with this one related to support for keyframes.

I updated this comment because the use of the word "Conditional" here was throwing me off every time I searched for it in the code. Could be worth investigating further, but I think it would go beyond the scope of this work.